styles/footer.less

```
@@ -0,0 +1,84 @@
+.footer
+{
```

стоит придерживаться одного стайлгайда в проекте.
```suggestion
.footer {
```

---

Что с форматированием? все осталось как и было

---

Только сейчас увидел :) 
Опустил все скобки в  проекте :) 
Если появился вопрос почему, ответ ниже:


Я всегда переношу скобки, так как являюсь противником оставлять скобки на той же  строчке, потому что в случае большого кода легче найти открывающую скобку , чем слово, особенно актуально, если зашел быстро подправить код в какой-нибудь блокнот где нет подсветок . И к тому же это никак не ухудшает код, просто добавляет строчку и все :) 
Обычно в проектах стоит eslint который перед пушем, поднимет или опустит согласно стилю команду, но в нашем случае, я так понял мы можем задать свой стиль